src/applications/disability-benefits/all-claims/utils/form0781.js

```diff
@@ -27,19 +27,15 @@ function combatOnlySelection(formData) {
  * @returns
  *   TRUE if
  *     - is set on form via the backend
- *     - Veteran is claiming a new disability
- *     - Veteran has selected connected condition choices on 'screener page'
+ *     - Veteran is claiming a new disability, and not a claim for increase
  *   else
  *     - returns false
  */
 export function showForm0781Pages(formData) {
-  const conditions = formData?.mentalHealth?.conditions || {};
   return (
     formData?.syncModern0781Flow === true &&
     isClaimingNew(formData) &&
-    Object.entries(conditions).some(
-      ([key, value]) => key !== 'none' && value === true,
-    )
+    !isClaimingIncrease(formData)
```

Comment:
Note: `isClaimingNew` and `isClaimingIncrease` should be mutually exclusive, but as you can see [here](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/disability-benefits/all-claims/utils/index.jsx#L334), `isClaimingNew` has a forced default value based on rated disabilities. To ensure the logic here does what we want, I've ensured `isClaimingIncrease` is always false

Reply:
If not claiming new, then wouldn't there not be any conditions to associate to mental health-related issues?